test/testCrossEntropy.jl

```diff
@@ -0,0 +1,20 @@
+# Tests concerning cross-entropy
+
+arr21 = rand(100)
+arr22 = rand(100)
+arr41 = rand(10000)
+arr42 = rand(10000)
+
+# The cross-entropy between two copies of an array
+@test cross_entropy(arr41, arr41) \approx log2(100) atol=0.01
```

Comment:
Julia 0.5 doesn't recognise this test syntax, so could you put these tests in the following format:
@test_approx_eq_eps cross_entropy(arr41, arr41) log2(100) 0.01
(You get deprecation warnings in Julia 0.6 but it passes)